games-misc/exult-sound/exult-sound-0-r2.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Sound data for games-engines/exult"
+HOMEPAGE="http://exult.sourceforge.net/"
+SRC_URI="mirror://sourceforge/exult/exult_audio.zip"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="!<games-engines/exult-9999"
+
+S="${WORKDIR}"
+
+src_install() {
+	insinto /usr/share/exult/music
+	doins "${WORKDIR}/music/"*ogg
```

Comment:
No need to reference WORKDIR all the time, just do `doins music/*.ogg` and likewise below.